client/src/styles/_variables.scss

```diff
@@ -12,10 +12,12 @@ $color-alabaster: #fafafa;
 $color-gallery: #eeeeee;
 $break-small: 420px;
 $break-medium: 768px;
+$break-md: 992px;
```

Comment:
why do we need one more breakpoint? and how do u calculate it?

Reply:
Volodya added md bootstrap class as a breakpoint and it is exactly 992 px